lib/services/elasticsearch.js

```diff
@@ -598,6 +611,10 @@ class ElasticSearch extends Service {
       [esRequest.type]: {}
     };
 
+    if (!this.kuzzle.indexCache.exists(esRequest.index)) {
+      return Bluebird.reject(new PreconditionError(`Index '${esRequest.index}' doesn't exist`))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)